packages/headless/src/commerce.index.ts

```diff
@@ -71,6 +77,8 @@ export type {
   InteractiveProductProps,
 } from './controllers/commerce/core/product-list/headless-core-interactive-product';
 
+export type {InteractiveResultCore} from './controllers/core/interactive-result/headless-core-interactive-result';
```

Comment:
Why do we need to export this? `InteractiveProduct` (which is exported above) is the type used for an interactive product controller.